### ISSUE TITLE
Rendering glitch seen on MacOS

### ISSUE THREAD

**Issue by chrisben** [boydm/scenic#112]
Rendering glitch shows up from time to time with the example app (scenic_new repo), sensor scene, :temperature text component.

### Checklist

- [X] Check other issues and make sure that it is not reported yet.

### Versions and Environment

Elixir:

```bash
# elixir -v
Erlang/OTP 21 [erts-10.1.2] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [hipe] [dtrace]

Elixir 1.7.4 (compiled with Erlang/OTP 21)
```

Erlang:

```bash
# erl -v
Erlang/OTP 21 [erts-10.1.2] [source] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1] [hipe] [dtrace]

Eshell V10.1.2  (abort with ^G)
```

Scenic:

```text
# mix deps | grep scenic
* scenic_sensor 0.7.0 (Hex package) (mix)
  locked at 0.7.0 (scenic_sensor) a12aed75
* scenic 0.9.0 (Hex package) (mix)
  locked at 0.9.0 (scenic) 129594f9
* scenic_driver_glfw 0.9.0 (Hex package) (mix)
  locked at 0.9.0 (scenic_driver_glfw) 4c7a8300
* scenic_clock 0.8.0 (Hex package) (mix)
  locked at 0.8.0 (scenic_clock) fc144430
```

OS:

```text
MacOS Mojave 10.14.1 
```

### Steps to reproduce

* Create the scenic example using `mix scenic.new.example`
* Compile and run
* Stay on the sensor scene, do nothing special
* Glitch mostly when temperature between 72.7° and 73°

### Expected Behavior

It should show the temperature as usual. Here's what it looks like before the issue:

<img width="566" alt="capture d ecran 2018-11-17 a 21 36 47" src="https://user-images.githubusercontent.com/679230/48665557-03489700-eab1-11e8-8377-5bf510f03160.png">

### Actual Behavior

But then his happens (here when displaying 72.8°):
 
<img width="561" alt="capture d ecran 2018-11-17 a 21 25 55" src="https://user-images.githubusercontent.com/679230/48665540-9fbe6980-eab0-11e8-9c94-aadff8db4591.png">
It looks like texts around it get mixed up.

It lasts only a few milliseconds (captured from a video recording of the screen), then things go back to normal.

I can reproduce the issue most of the times.

#### Stack Trace

n/a

### Additional Comments

I checked with IO.inspect the text given to the component (temperature), it's correct. Looks like the issue is from scenic, not the example project itself.

**Comment by boydm** [comment]
This is almost certainly in the glfw driver (and won't be easy to fix). Is on the list tho.

**Comment by boydm** [comment]
This was probably fixed earlier and is almost definitely fixed in v0.11
